src/components/MovieCard/MovieCard.tsx

```diff
@@ -2,6 +2,49 @@ import React from 'react';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  title: string;
+  description: string;
+  imgUrl: string;
+  imdbUrl: string;
+  imdbId: string;
+};
+
+export const MovieCard: React.FC<Props> = ({
+  title,
+  description,
+  imgUrl,
+  imdbUrl,
+  imdbId,
+}) => (
+  <div className="card" data-cy="card" key={imdbId}>
```

Comment:
The key always should be in map function call, [reactjs.org/docs/lists-and-keys.html#extracting-components-with-keys](https://reactjs.org/docs/lists-and-keys.html#extracting-components-with-keys)